app/models/user.rb

```diff
@@ -76,4 +78,8 @@ def create_activation_digest
     self.activation_token = User.new_token
     self.activation_digest = User.digest(activation_token)
   end
+
+  def remove_orders
+    Order.where(user_id: id).destroy_all
```

Comment:
do you need here `user_id: current_user.id`? If not, where does that `id` come from?

Reply:
from self.id?